students/gyuchan/users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path, include
+
+from .views import SignUpView
+
+urlpatterns = [
+    path('', SignUpView.as_view())
```

Comment:
현재 작성한대로면 endpoint 주소가 `:8000/users` 입니다.
따라서 기능에 맞게 수정해주세요.
```suggestion
    path('/signup', SignUpView.as_view())
```